src/encoded/static/components/award.js

```diff
@@ -113,7 +141,11 @@ function createDoughnutChart(chartId, values, labels, colors, baseSearchUri, nav
                         if (activePoints[0]) { // if click on wrong area, do nothing
                             const clickedElementIndex = activePoints[0]._index;
                             const term = chart.data.labels[clickedElementIndex];
-                            navigate(`${baseSearchUri}${globals.encodedURIComponent(term)}`);
+                            if (chart.options.onClick.baseSearchUri) {
```

Comment:
It gets set when the chart gets updated because of user input, but this is a highly unusual way to save data, though technically it works. At the very least you need to comment it like crazy because adding properties to a function is highly unusual. I’m not sure why you needed this, making the baseSearchUri sometimes unused.